src/components/Content/Page.js

```diff
@@ -12,6 +12,9 @@ class Page extends React.Component {
     /* We can insert our html here directly since we trust our backend cms */
     return (
       <div>
+        <div>
+          <h1 className={style.heading}>{this.props.page.title}</h1>
+        </div>
```

Comment:
You don't need to wrap the `<h1>` in a `<div>`. You can just replace
```html
<div>
  <h1 className={style.heading}>{this.props.page.title}</h1>
</div>
```
with
```html
  <h1 className={style.heading}>{this.props.page.title}</h1>
```
It would also be cool, if you create a simple component `Heading` which gets a string `title` as a property and displays just this h1 snippet with the correct styling. Then we can use that component on the Content Page, the LandingPage and so on, and have the styling in one place.

Reply:
i called it Caption since there is already a Heading component on the LocationPage